src/containerapp/azext_containerapp/commands.py

```diff
@@ -176,15 +176,27 @@ def load_command_table(self, args):
 
     with self.command_group('containerapp env java-component', is_preview=True) as g:
         g.custom_command('list', 'list_java_components')
-
+    
     with self.command_group('containerapp env java-component spring-cloud-config') as g:
```

Comment:
You need to add deprecated_info in command group and commands.
For example:
```
    with self.command_group('containerapp service postgres', deprecate_info=self.deprecate(redirect='containerapp add-on postgres', hide=True)) as g:
        g.custom_command('create', 'create_postgres_service', supports_no_wait=True)
        g.custom_command('delete', 'delete_postgres_service', confirmation=True, supports_no_wait=True)
```

Reply:
added deprecated_info in command group